modules/ROOT/pages/troubleshoot-saml-assertions-task.adoc

```diff
@@ -1,9 +1,12 @@
-= To view a SAML response in your Browser
+= View a SAML Response in the Browser
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Follow this procedure to help the MuleSoft Support team to assist you in troubleshooting your SAML 2.0 compliant SSO setup or to just view your SAML response.
+To view your SAML response in a browser, perform a single sign-on and use the developer tools in your browser to find the bearer token and SAML response.
+
+[NOTE]
+Complete this task before calling the MuleSoft Support team to assist you in troubleshooting your SAML 2.0 compliant SSO setup. 
 
 . Open a new tab in your browser and use Chrome Developer Tools, Firebug for Firefox, or a SAML tracer to be prepared to capture the SAML response. Ensure that you are preserving logs.
```

Comment:
Line 11 should probably just be "to prepare to capture" instead of "to be prepared"
How do you ensure you are preserving logs?